esmvalcore/cmor/_fixes/native6/era5.py

```diff
@@ -0,0 +1,242 @@
+"""Fixes for ERA5."""
+import numpy as np
+import iris
+
+from ..fix import Fix
+from ..shared import add_scalar_height_coord
+
+
+class FixEra5(Fix):
+    """Fixes for ERA5 variables."""
+
+    @staticmethod
+    def _frequency(cube):
+
+        if not cube.coords(axis='T'):
+            return 'fx'
+        time = cube.coord(axis='T')
+        if len(time.points) == 1:
+            return 'fx'
+
+        interval = time.points[1] - time.points[0]
+        unit = 'hours' if 'hour' in time.units.name else 'days'
+        if (unit == 'hours' and interval == 1):
+            return 'hourly'
+        if (unit == 'days' and interval - 1 / 24 < 1e-4):
+            return 'hourly'
+        return 'monthly'
+
+
+class Accumulated(FixEra5):
+    """Fixes for accumulated variables."""
+
+    def _fix_frequency(self, cube):
+        if cube.var_name in ['mx2t', 'mn2t']:
```

Comment:
Could you put the cmor variable names in a comment? That will make the code easier to understand.